definitions/variable/afolu/land-cover-change.yaml

```diff
@@ -0,0 +1,65 @@
+- Forest Area Change:
+    description: Annual net change in total forest area between last period and this 
+      period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation:
+    description: Annual gross deforestation between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation|Primary:
+    description: Annual gross loss of primary forest between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation|Secondary:
+    description: Annual gross loss of secondary forest between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion:
+    description: Annual gross forest expansion between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion|Secondary:
+    description: Annual gross expansion of secondary forest between last 
+      period and this period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion|Planted:
+    description: Annual gross expansion of planted forest between last period and this 
+      period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion|Planted|Plantation:
+    description: Annual gross expansion of plantations between last period and this 
+      period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion|Planted|Plantation|Timber:
+    description: Annual gross expansion of timber plantations between last period and this 
+      period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion|Planted|Plantation|Re-Afforestation:
```

Comment:
It's not clear to me what Re-Afforestation is, in particular if it's a companion to "timber plantations", but then described as carbon plantation...

Reply:
revised definition: Annual gross expansion of carbon plantations; Re-Afforestation with monoculture and/or non-native species